File: d066(go_to_school)/C++/go_to_school.cpp
#include <iostream>

int main(void) {
  int hour, min;
  while (std::cin >> hour >> min) {
    if (hour >= 17 || hour < 7 || (hour < 8 && min < 30))
      std::cout << "Off School" << '\n';
    else std::cout << "At School" << '\n';
  }
  return 0;
}

File: d066(go_to_school)/Python/go_to_school.py
from sys import stdin, stdout
while True:
    line = stdin.readline()
    if not line: break
    hour, min = map(int, line.split())
    if hour >= 17 or hour < 7 or (hour < 8 and min < 30):
        stdout.write("Off School\n")
    else: stdout.write('At School\n')
